test/wolff_tests.erl

```diff
@@ -477,17 +481,22 @@ test_message_too_large() ->
       Self = self(),
       AckFun = {fun ?MODULE:ack_cb/4, [Self, Ref]},
       _ = wolff:send(Producers, Batch, AckFun),
-      ?WAIT(5000, {ack, Ref, _Partition, BaseOffset}, BaseOffset)
+      fun() -> ?WAIT(5000, {ack, Ref, _Partition, BaseOffset}, BaseOffset) end
```

Comment:
Nit: this wrapping seems unnecessary.

Reply:
It's necessary for the SendFuc to be async. Can return Ref and then do `?WAIT` outside, but it repeats the tuple elements for each macro.